### DISCUSSION TITLE
Is the organization free subscription free?

### DISCUSSION THREAD

**Discussion by d3stiny-io** [community/community#136627] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was wondering if its free

**Comment by ferid333** [comment]
Can you ask question in detail ?

**Reply by d3stiny-io** [reply]
When i want to make the organization for something and i see the plans like there's free and others

**Reply by ferid333** [reply]
You can register as a student and it will give free trials for certain time

**Reply by d3stiny-io** [reply]
But i didn't register as a student

**Reply by ferid333** [reply]
If you want free trial you can register with email ending with .edu
